src/locales/en_US.js

```diff
@@ -675,7 +677,10 @@ const strings = {
   fio_address_details_screen_alert_button: 'View Details',
   fio_address_details_screen_registered: 'Registered',
   fio_address_details_screen_expires: 'Expires',
+  fio_address_details_screen_bundles: 'Remaining bundles',
   fio_address_details_screen_manage_account_settings: 'Address Settings',
+  fio_address_details_bundles_out_soon: "Your FIO address's bundled transactions is almost spent. Would you like to purchase more bundles?",
```

Comment:
"You have spent almost all the bundled transactions in your FIO address. Would you like to purchase more bundled transactions?"